src/sagas/repository.ts

```diff
@@ -540,23 +540,6 @@ export function* removeDatasetPolicyMember({ payload }: any): any {
   }
 }
 
-export function* getDatasetTablePreview({ payload }: any): any {
-  const { dataset, tableName } = payload;
-  const datasetProject = dataset.dataProject;
-  const datasetBqSnapshotName = `datarepo_${dataset.name}`;
-  const url = `/bigquery/v2/projects/${datasetProject}/datasets/${datasetBqSnapshotName}/tables/${tableName}/data`;
-  try {
-    const response = yield call(authGet, `${url}?maxResults=100`, true);
-    yield put({
-      type: ActionTypes.GET_DATASET_TABLE_PREVIEW_SUCCESS,
-      preview: response,
-      tableName,
-    });
-  } catch (err) {
-    showNotification(err);
-  }
-}
-
```

Comment:
Change 3 - Drive by change - as far as I can tell, we're not using this anywhere.